### PR TITLE
Set up performance test infra

### PR DESCRIPTION
**Pull request by chizhg**
Set up performance test infra.

In place of https://github.com/google/knative-gcp/pull/328

/cc @nachocano 
/cc @grantr

### CODE REVIEW

File: test/performance/benchmarks/broker-pubsub/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 The Knative Authors
```
Comment (nachocano) [inline test/performance/benchmarks/broker-pubsub/cluster.yaml]: update copyright, we are using `Copyright 2019 Google LLC` in this repo... 
here an in the other files as well

File: test/performance/benchmarks/broker-pubsub/dev.config [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+benchmark_name: "Development - PubSub Broker Latency & Throughput"
+description: "Measure latency and throughput of the broker using various channels."
+benchmark_key: '6592390798245888'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "grantrodgers@google.com"
+owner_list: "chizhg@google.com"
+owner_list: "xiyue@google.com"
+owner_list: "gracegao@google.com"
+owner_list: "nachocano@google.com"
```
Comment (nachocano) [inline test/performance/benchmarks/broker-pubsub/dev.config]: can you add chen here as well. I think he is in the eventing continuous already...

File: test/performance/benchmarks/broker-pubsub/dev.config [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+# Anyone can add their IAM robot here to publish to this benchmark.
+owner_list: "mako-job@knative-performance.iam.gserviceaccount.com"
+# This is grantrodgers' robot:
+owner_list: "mako-upload@grantrodgers-crd.iam.gserviceaccount.com"
+owner_list: "mako-upload@xiyue-knative-project.iam.gserviceaccount.com"
+owner_list: "mako-upload@gracegao-knative-gcp-testing.iam.gserviceaccount.com"
+owner_list: "mako-upload@knative-project-228222.iam.gserviceaccount.com"
```
Comment (nachocano) [inline test/performance/benchmarks/broker-pubsub/dev.config]: same here, I think he has his own mako-upload service account as well

File: test/performance/benchmarks/channel-pubsub/dev.config [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+benchmark_name: "Development - PubSub Channel Latency & Throughput"
+description: "Measure latency and throughput of channels."
+benchmark_key: '6246352530964480'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "grantrodgers@google.com"
+owner_list: "chizhg@google.com"
+owner_list: "xiyue@google.com"
+owner_list: "gracegao@google.com"
+owner_list: "nachocano@google.com"
```
Comment (nachocano) [inline test/performance/benchmarks/channel-pubsub/dev.config]: same for this file. Can you add Chen?

File: test/performance/benchmarks/channel-pubsub/continuous/200-channel-perf.yaml [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            role: channel-perf-consumer
+        spec:
+          serviceAccountName: perf-pubsub
+          restartPolicy: Never
+          containers:
+            - name: sender-receiver
+              image: knative.dev/eventing/test/test_images/performance
+              args:
+                - "--roles=sender,receiver"
+                # FIXME: not sure if this address is correct
```
Comment (nachocano) [inline test/performance/benchmarks/channel-pubsub/continuous/200-channel-perf.yaml]: I think this is correct, remove the FIXME

**Comment by nachocano** [comment]
Thanks for doing this @chizhg !!! Just a few very cosmetic things...

File: test/performance/benchmarks/broker-pubsub/cluster.yaml (hunk 1 above)
Comment (chizhg) [reply test/performance/benchmarks/broker-pubsub/cluster.yaml]: Done.

File: test/performance/benchmarks/broker-pubsub/dev.config (hunk 2 above)
Comment (chizhg) [reply test/performance/benchmarks/broker-pubsub/dev.config]: Added.

File: test/performance/benchmarks/broker-pubsub/dev.config (hunk 3 above)
Comment (chizhg) [reply test/performance/benchmarks/broker-pubsub/dev.config]: Done.

File: test/performance/benchmarks/channel-pubsub/dev.config (hunk 4 above)
Comment (chizhg) [reply test/performance/benchmarks/channel-pubsub/dev.config]: Done.

File: test/performance/benchmarks/channel-pubsub/continuous/200-channel-perf.yaml (hunk 5 above)
Comment (chizhg) [reply test/performance/benchmarks/channel-pubsub/continuous/200-channel-perf.yaml]: Removed.

**Pull request merged**